src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/Example3.java

```diff
@@ -8,10 +8,15 @@
   </module>
 </module>
 */
+//non-compiled with javac: Compilable with Java14
 
 // xdoc section -- start
-public void myTest() {
-    final String quote = """
-               \""""; // violation
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltokentext;
+
+public class Example3 {
+    public void myTest() {
```

Comment:
please make indentation as +2 for all examples files to make it shorter and result web pages will be easier to review on small screen.

Reply:
Done